### ISSUE TITLE
Parallel Uploads in FPP Connect

### ISSUE THREAD

**Issue by MrPaulAR** [xLightsSequencer/xLights#2906]
**Describe the solution you'd like**
Instead of doing sequential uploads to all devices (Kulp/Falcon) via FPP Connect send them in parallel (configurable via settings).  For those of us with multiple devices that support this feature and larger shows it'll greatly improve upload times.

It would still process one song at a time but just do the upload portion.

- Sequence 1
  - Copy audio
  - Generate FSEQ file(s)
  - async upload FSEQ files
  - await completion of uploads
- Sequence 2
  - Repeat above
- Etc...

**Additional context**

**Comment by kwoodard** [comment]
I think this is a duplicate of #2351.  I had a similar thought and have been looking at the FPP Connect code.  While it seems possible to parallelize this, it seems the management of the progress dialog boxes might actually be the most challenging part of this request.
